order.c

```diff
@@ -0,0 +1,45 @@
+//
+// Created by ileana on 11/6/2019.
+//
+
+#include<stdio.h>
+#include<string.h>
+void displayFoodData(char food[],double priceFood, char drink[], double priceDrink, char username[]){
+    printf("This is your order:\n");
+    printf("-------------------\n");
+    printf("Name: %s\n", username);
+    printf("Food items:\n");
+    printf("---%s: (%0.2f)\n",food,priceFood);
+    if(priceDrink!=0) {
```

Comment:
why not display it if 0?

Reply:
In my code if the priceDrink is equal to 0 that means that the user chose the option "No, thanks!" so he does not want a drink. I was thinking that I should not display it on the final order if he doesn't choose a specific drink, similarly to the case of cutlery and addition information options. However, if  the priceDrink is different from 0, therefore the user chose a drink, it will be displayed on the final order.

Reply:
seemed a bit hacky, but it's fine